README.md

```diff
@@ -134,6 +130,12 @@ Mods that make other mods work. Hidden from in-game mod menu, but should not be
 | [Indium](https://www.curseforge.com/minecraft/mc-mods/indium) | [comp500](https://www.curseforge.com/members/comp500) | &nbsp;&nbsp;✔️ | &nbsp;&nbsp;✔️ | &nbsp;&nbsp;✔️ | Adds support for the Rendering API (more parity mods) |
 | [MidnightLib](https://www.curseforge.com/minecraft/mc-mods/midnightlib) | [TeamMidnightDust](https://www.curseforge.com/members/TeamMidnightDust) |  |  | &nbsp;&nbsp;✔️ | Makes some mods work |
 
+## Notes
+
+¹ [Hydrogen is not from CurseForge](https://support.overwolf.com/en/support/solutions/articles/9000197913-non-curseforge-mods),
+
+² Colormatic currently has problems, it will be re-added once resolved.
+

```

Comment:
not so sure about that, it can be found better when it is right below the table

Reply:
I seperated them from the table sections as i cannot find a neater way to put these notes elsewhere, i could go ahead and move them back but then the columns would not be quite as clean anymore.

Reply:
okay, show me what magic you can do with the columns first and then we'll see where to put them

Reply:
Marking this as resolved as multiple attempts were made but to no avail - this change above will have to remain sadly.